exchanges/bittrex/bittrex_test.go

```diff
@@ -604,7 +604,7 @@ func TestModifyOrder(t *testing.T) {
 	}
 }
 
-func WithdrawCryptocurrencyFunds(t *testing.T) {
+func WithdrawCryptocurrencyFunds(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting